Library/Formula/postgresql.rb

```diff
@@ -77,8 +82,16 @@ def install
   end
 
   def post_install
-    unless File.exist? "#{var}/postgres"
-      system "#{bin}/initdb", "#{var}/postgres"
+    system "pkill postgres || true"
+
+    if (var/"postgres").exist?
+      mv var/"postgres", var/"postgres-#{version_suffix}"
+    elsif (var/"postgres-#{version_suffix}").exist?
+      puts ""
```

Comment:
Remove the preceding two lines and make the following line an `if` rather than `elsif`.